services/metadata/backend/data/digitalagenda.json

```diff
@@ -0,0 +1,416 @@
+            "13.\tGuerrero Cantarell, Rosalía. “Technology as a Woman’s Call: The Efforts of the Fredrik Bremer Association to Promote Women’s Education in Technology 1978–1999.” Nordic Journal of Educational History. Special Issue: Exploring the History of Knowledge and Education 9, no. 2 (2022): 125–147. https://doi.org/10.36368/njedh.v9i2.268.",
+            "14.\tGuerrero, Rosalía, Carmen Flury and Michael Geiss. “From Victims to Economic Assets: Training Women in an Emerging Digital Society During the Late 1970s to the Mid-1990s.” History of Media Studies 3 (July 2023). https://doi.org/10.32376/d895a0ea.6e09b010."
+        ],
+        "name": "Education and the European Digital Agenda: Switzerland, Germany and Sweden after 1970",
+        "shortcode": "0848",
+        "spatialCoverage": [
+            {
+                "__type": "URL",
+                "type": "Geonames",
+                "url": "https://www.geonames.org/2658434/switzerland.html",
+                "text": "Switzerland"
+            },
+            {
+                "__type": "URL",
+                "type": "Geonames",
+                "url": "https://www.geonames.org/11612754/bundesrepublik-deutschland-bis-1990.html",
+                "text": "Bundesrepublik Deutschland (bis 1990)"
+            },
+            {
+                "__type": "URL",
+                "type": "Geonames",
+                "url": "https://www.geonames.org/2661886/kingdom-of-sweden.html",
+                "text": "Sweden"
+            },
+            {
+                "__type": "URL",
+                "type": "Geonames",
+                "url": "https://www.geonames.org/6695072/european-union.html",
+                "text": "European Union"
+            },
+            {
+                "__type": "URL",
+                "type": "Geonames",
+                "url": "https://www.geonames.org/2921044/federal-republic-of-germany.html",
+                "text": "Germany"
+            },
+            {
+                "__type": "URL",
+                "type": "Geonames",
+                "url": "https://www.geonames.org/8354410/german-democratic-republic.html",
+                "text": "East Germany"
+            }
+        ],
+        "startDate": "2019-04-01",
+        "temporalCoverage": [
+            {
+                "en": "1970–2000"
+            }
+        ],
+        "url": {
+            "__type": "URL",
+            "type": "URL",
+            "url": "https://app.dasch.swiss/project/23m2HHBlR5GXhAu7rXPRmw",
+            "text": "Discover Project Data"
+        },
+        "secondaryURL": {
+            "__type": "URL",
+            "type": "URL",
+            "url": "https://www.digitalagenda.ch/",
+            "text": "External Project Website"
+        }
+    },
+    "datasets": [
+        {
+            "__id": "http://ns.dasch.swiss/repository#dsp-0848-dataset-000",
```

Comment:
This link does not work

Reply:
it's an internal link which is not relevant for the user